DemoServer/Controllers/Demos/Spatial/SpatialIndex/metadata.json

```diff
@@ -0,0 +1,271 @@
+                "  These fields will be indexed and can be queried on.",
+                "",
+                "* Field _LocationCoordinates_ will be configured as a spatial field holding the spatial data from the document (see next step)."
+            ],
+            "Assets": [
+                {
+                    "Title": "Map Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/map-indexes"
+                },
+                {
+                    "Title": "Static Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "The index Map",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* Define the index-fields within the Map function.",
+                "",
+                "* Call `CreateSpatialField()` to create the field as a spatial field.<br>",
+                "  Pass the path to:<br>",
+                "  * the fields that contain the latitude & longitude data in the document, or",
+                "  * the field that contains the WKT shape string in the document.",
+                "",
+                "* The spatial field value will be stored on the index."
+            ],
+            "Assets": [
+                {
+                    "Title": "Map Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/map-indexes"
+                },
+                {
+                    "Title": "Spatial indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#spatial-indexes"
+                },
+                {
+                    "Title": "Index Spatial Data",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/indexing-spatial-data"
+                },
+                {
+                    "Title": "Store fields on index Demo",
+                    "Type" : "demo",
+                    "Url"  : "/static-indexes/store-fields-on-index"
+                }
+            ]
+        },
+        {
+            "Title": "Customize spatial indexing strategy",
+            "Slug" : "step-4",
+            "DescriptionLines": [
+                "* Use `Spatial()` to customize the strategy by which the spatial field will be indexed.",
```

Comment:
I will add text to this step saying that it is Not mandatory to customize the indexing strategy.
And in that case the default strategy is Geohash with 9 tree level.

So you don't have to call this `Spatial()` method.
Field `LocationCoordinates` **is** a spatial field because it was created a step above with
`CreateSpatialField`